<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProperty</class>
 <widget class="QDialog" name="AddProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1981</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancle">
   <property name="geometry">
    <rect>
     <x>1890</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1981</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="box1"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="box2"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="box3"/>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="box4"/>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="box5"/>
    </item>
    <item row="0" column="5">
     <widget class="QComboBox" name="box6"/>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="box7"/>
    </item>
    <item row="0" column="7">
     <widget class="QComboBox" name="box8"/>
    </item>
    <item row="0" column="8">
     <widget class="QComboBox" name="box9"/>
    </item>
    <item row="0" column="9">
     <widget class="QComboBox" name="box10"/>
    </item>
    <item row="0" column="10">
     <widget class="QComboBox" name="box11"/>
    </item>
    <item row="0" column="11">
     <widget class="QComboBox" name="box12"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
